src/patternfly/components/Tabs/tabs.scss

```diff
@@ -3,6 +3,7 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
 .pf-c-tabs {
   --pf-c-tabs--inset: 0;
```

Comment:
wdyt about adding `.pf-m-page-insets` to tabs the way we have that for toolbar? Also I wonder if that should be `pf-m-inset-page`?